spec/views/events/show.html.erb_spec.rb

```diff
@@ -64,6 +64,33 @@
     expect(rendered).to have_link(t(:print_button_label, scope: 'events.badges'))
   end
 
+  it "should not display accept-all-button for non-organizers" do
+    [:coach, :student].each do | each |
+      sign_in(FactoryGirl.create(:user, role: each))
+      render
```

Comment:
Dont you need to set the max participants to unlimited before this line? Otherwise this test could be green just because there are not enough places